ext/phar/phar_object.c

```diff
@@ -5239,6 +5239,41 @@ ZEND_END_ARG_INFO()
 
 #endif /* HAVE_SPL */
 
+/* PharData share the implement of Phar class, override are implemented inside those methods */
```

Comment:
Hi @smalyshev, It is not the whole function table, this function table contains only functions for `PharData` class. (from the doc: http://php.net/phardata), those method internally handle different classes like this https://github.com/php/php-src/blob/master/ext/phar/phar_object.c#L1126 .